liquibase-standard/src/test/groovy/liquibase/change/core/LoadDataChangeTest.groovy

```diff
@@ -706,6 +706,9 @@ public class LoadDataChangeTest extends StandardChangeTest {
         assert dCol2 instanceof Timestamp
         ((Timestamp) dCol2).toLocalDateTime() == LocalDateTime.parse("2020-01-02T03:04:05")
         columnValue(sqlStatements[2], Col.bool) == Boolean.TRUE
+        for (Col n : [Col.name, Col.id, Col.date, Col.bool]) {
```

Comment:
new tests?

Reply:
Do you mean adding a test cover for this case separately? I just updated that test that it's verifying some different test data cases, and as I added a new input line I added this check. I can take it to a separate case, although I think the ideal would be to refactor this test case.

Reply:
No, that's fine!